app/helpers/application_helper.rb

```diff
@@ -22,12 +22,22 @@ def markdown(text)
     (redcarpet.render text).html_safe
   end
 
-  def up_vote_link_classes(argument)
-
+  def up_vote_link_classes(post)
+    class_start = "glyphicon glyphicon-chevron-up "
+    if current_user.voted(post) && current_user.voted(post).up_vote?
+      class_start + 'voted'
+    else
+      class_start + ''
+    end
   end
 
-  def down_vote_link_classes(argument)
-    
+  def down_vote_link_classes(post)
+    class_start = "glyphicon glyphicon-cheveron-down "
+    if current_user.voted(post) && current_user.voted(post).down_vote?
+      class_start + 'voted'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.